### PR TITLE
fix ldap ui buttons

### LINKED ISSUE #33866: hover on legacy buttons looks strange and makes elements juggle
See 

https://user-images.githubusercontent.com/42591237/188123540-67ff37f9-585b-4f0e-ac2f-af338c06e5b2.mp4

### PR DESCRIPTION
**Pull request by szaimen**
Close https://github.com/nextcloud/server/issues/33866

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Review by blizzz — APPROVED** [review]

File: core/src/jquery/css/jquery-ui-fixes.scss [hunk 1]
```diff
@@ -228,6 +228,11 @@
 	border: 1px solid var(--color-primary-text);
 }
 
+// fix ui-buttons on hover
+.ui-button:hover {
+    font-weight:bold !important;
```
Comment (Pytal) [inline core/src/jquery/css/jquery-ui-fixes.scss:233]:
```suggestion
    font-weight: bold !important;
```
**suggestion (non-blocking)**

**Review by Pytal — APPROVED** [review]

File: core/src/jquery/css/jquery-ui-fixes.scss (hunk 1 above)
Comment (szaimen) [reply core/src/jquery/css/jquery-ui-fixes.scss:233]: thanks but will not do this as it is just too much effort to push, recompile and wait for the CI

**Comment by szaimen** [comment]
CI failure unrelated

**Pull request merged**